scheduler/src/main/resources/public/partials/cluster.html

```diff
@@ -27,11 +27,11 @@ <h3 class="font-red-haze">{{configuration.NumberOfHwNodes}}</h3>
             <!--</div>-->
         </div>
     </div>
-    <div class="col-md-3">
+    <div class="col-md-4">
         <div class="cell dashboard-stat">
             <div class="display">
                 <div class="number">
-                    <h3 class="font-red-haze">{{configuration.Mem}}{{configuration.MemUnits}}</h3>
+                    <h3 class="font-red-haze">{{configuration.Mem}}MB</h3>
```

Comment:
Ah, yeah, sorry I broke this. But there is no related Mesos concept of units, do I deleted it.